src/constructor.io/models/Quizzes/QuizRequest.cs

```diff
@@ -24,7 +24,12 @@ public class QuizRequest
         /// <summary>
         /// Gets or sets version identifier for the quiz.
```

Comment:
Should we make this more verbose with an explanation about when to use this?

https://github.com/Constructor-io/constructorio-client-javascript/pull/245#discussion_r1150842443

Reply:
Totally, I missed out what we did for client-js but it's definitely nice to keep things consistent across SDKs.